src/App.tsx

```diff
@@ -1,26 +1,46 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
```

Comment:
Ensure that the `getTodos` function is correctly defined in the `./api` module and returns a promise that resolves to an array. This is crucial for the `.then` method used later in the code.